src/planscape/plan/views.py

```diff
@@ -270,3 +280,47 @@ def get_project_areas(request: HttpRequest) -> HttpResponse:
         return JsonResponse(response)
     except Exception as e:
         return HttpResponseBadRequest("Ill-formed request: " + str(e))
+
+
+def get_scores(request: HttpRequest) -> HttpResponse:
+    try:
+        plan = get_plan_by_id(request.GET)[0]
+        geo = plan.geometry
+        reg = plan.region_name.removeprefix('RegionName.').lower()
+
+        accumulator = RasterPixelAccumulator(geo)
+
+        ids_to_conditions = {
+            c.id: c.condition_name
+            for c in BaseCondition.objects.filter(region_name=reg).all()}
+        ids_to_raster_names = {
+            c.condition_dataset_id: c.raster_name
+            for c in Condition.objects.filter(is_raw=False).all()}
```

Comment:
Do we also want the raw scores?
And if so, should the preference be set via a url parameter, or should both normalized and raw scores be returned?

Reply:
I think this is a product question. During planning, do we want to show normalized or raw scores for the planning area? @mskim3494 @ayee07

Reply:
noted.
I'll submit the version with normalized scores for now.
I suspect raw vs normalized will be a url param but will wait for Minsu and Alex to comment.